addon/components/basic-dropdown-optional-tag.ts

```diff
@@ -0,0 +1,5 @@
+import Component from '@glimmer/component';
+
+interface BasicDropdownOptionalTagArgs {}
+
+export default class BasicDropdownOptionalTag extends Component<BasicDropdownOptionalTagArgs> {}
```

Comment:
Could this be a template-only component?

Reply:
I wasn't quite sure how that would work in an app that hasn't enabled octane? would it then not be a "regular" ember-component with a wrapping element?

Reply:
I didn't consider apps with octane disabled. This looks good.